_pkgdown.yml

```diff
@@ -0,0 +1,7 @@
+url: https://epiverse-trace.github.io/vaccineff/
```

Comment:
A pkgdown structure for the reference page on the website would be useful for grouping functions.

Currently there are only three vignettes so you don't need to structure them but if you add more vignettes then this could also be useful.

See: https://pkgdown.r-lib.org/articles/customise.html#layout

Reply:
Thanks! I will take it into consideration in https://github.com/epiverse-trace/vaccineff/issues/88